app/assets/javascripts/models/beta/projectBoard.js

```diff
@@ -4,5 +4,12 @@ import camelCase from 'camelcase-object-deep';
 export function get(projectId) {
   return axios
     .get(`/beta/project_boards/${projectId}`)
-    .then(({ data }) => camelCase(data, {deep: true}));
+    .then(({ data }) => camelCase(data, {deep: true}))
+    .then(( projectBoard ) => ({
```

Comment:
What do you think about adding a comment explaining why this was necessary and that eventually we're gonna fix the backend and remove it from here?